README.md

```diff
@@ -13,19 +13,36 @@ system. On a Linux system this can be done using the command line
 
 For a more permanent change edit `/etc/sysctl.conf`.
 
-## Build
+## Install Dependencies and Build
 
-To build the charm, first install the `charmcraft` tool
+To build the charm, first install `charmcraft`,  `juju` and `microk8s`
 
-    sudo snap install charmcraft --classic
+    snap install charmcraft
+    snap install juju --classic
+    snap install microk8s --classic 
 
 Then in this git repository run the command
 
     charmcraft build
 
 ## Usage
 
-TODO: explain how to use the charm
+    juju deploy ./elasticsearch.charm
+
+To scale up:
+
+    juju add-units -n 2 elasticsearch
+
+> Note: When the total number of nodes in the cluster is 2, split brain is possible. If there are currently two nodes, be sure to use `juju add-units` to scale up to a functional HA cluster.
```

Comment:
Wonder if "split brain" should be linked to Elasticsearch documentation page explaining what it is and what the consequences are ?

Reply:
I like this idea. Will add.